src/browser/ChromeListener.cpp

```diff
@@ -112,8 +112,8 @@ void ChromeListener::readDatagrams()
 
 void ChromeListener::readHeader(boost::asio::posix::stream_descriptor& sd)
 {
-    char buf[4] = {};
-    async_read(sd, buffer(buf,sizeof(buf)), transfer_at_least(1), [&](error_code ec, size_t br) {
+    std::array<char, 4> buf;
+    async_read(sd, buffer(buf), transfer_at_least(1), [&](error_code ec, size_t br) {
         if (!ec && br >= 1) {
             uint len = 0;
             for (int i = 0; i < 4; i++) {
```

Comment:
Could you maybe use buf.size() here instead of a magic 4 or at least buf.at(i) in the next line instead of buf[i], so we have explicit bounds checks? This is just to be sure we won't introduce any errors in the future.